.github/workflows/build_and_test.yml

```diff
@@ -26,95 +20,35 @@ env:
 
 jobs:
   braket-checks:
-    name: Braket - Build and test module
+    name: Build and test module
     strategy:
       matrix:
         os: ['ubuntu-latest', 'macos-latest', 'windows-latest']
+        pyver: ['3.10', '3.12']
```

Comment:
Why not 3.11?

Reply:
I think it is enough to test the minimum and maximum supported versions.

I don't remember why we don't yet support 3.13 for this package, but will test to see if we can now.